iothub_service_client/src/iothub_messaging_ll.c

```diff
@@ -457,15 +457,16 @@ static char* createAuthCid(IOTHUB_MESSAGING_HANDLE messagingHandle)
     }
     else
     {
-        const char* AMQP_SEND_AUTHCID_FMT = "iothubowner@sas.root.%s";
-        size_t authCidLen = strlen(AMQP_SEND_AUTHCID_FMT) + strlen(messagingHandle->iothubName);
+        const char* AMQP_SEND_AUTHCID_FMT = "%s@sas.root.%s";
```

Comment:
>const char* AMQP_SEND_AUTHCID_FMT [](start = 8, length = 33)

i usually prefer using `const char[]`  because it allows the next line to be a `sizeof()` instead of needing to count the characters by hand.